ad-insertion/video-analytics-service/samples/sample.py

```diff
@@ -45,17 +45,35 @@ def print_json(object):
     
 def read_detection_results(destination,verbose=True):
     if (verbose):
+        object_lines=[]
         with open(destination) as file:
             for line in file:
-                print("Detection Result: \n")
-                print_json(json.loads(line))
-            
+                try:
+                    if (line=="{\n"):
+                        object_lines.append(line)
+                        line=None
+                        
+                    if (object_lines):
+                        if (line):
+                            object_lines.append(line)
+                        if (line=='}\n'):
+                            line="".join(object_lines)
+                            object_lines=[]
+                        else:
+                            line=None
+
+                    if (line):
+                        print("Detection Result: \n")
+                        print_json(json.loads(line))
+                except Exception as error:
+                    print(error)
+                
```

Comment:
No parens in 'if' statements

Reply:
don't necessarily like it - but can do it